bin/improver-wind-downscaling

```diff
@@ -122,15 +123,14 @@ def main():
         wind_speed_iterator = wind_speed.slices_over('realization')
     except CoordinateNotFoundError:
         wind_speed_iterator = [wind_speed]
-    wind_speed_list = iris.cube.CubeList()
-    for wind_speed_slice in wind_speed_iterator:
-        result = (
-            wind_downscaling.RoughnessCorrection(
-                silhouette_roughness_filepath, sigma, target_orog,
-                standard_orog, float(args.model_resolution),
-                z0_cube=veg_roughness_cube,
-                height_levels_cube=height_levels).process(wind_speed_slice))
-        wind_speed_list.append(result)
+
+    downscaler = wind_downscaling.RoughnessCorrection(
+            silhouette_roughness_filepath, sigma, target_orog,
+            standard_orog, float(args.model_resolution),
+            z0_cube=veg_roughness_cube,
+            height_levels_cube=height_levels).process
+
+    wind_speed_list = apply_plugin(downscaler, wind_speed_iterator)
```

Comment:
I've noticed that when running the CLI tests, the `wind-downscaling/04-with_realization.bats` test is failing with the error:
`RuntimeError: IOLoop is closed`. I believe this tests where the input file has a `realization` dimension coordinate, so the wind_speed_iterator is `wind_speed.slices_over("realization")`.